src/styles/blocks/catalog.scss

```
@@ -0,0 +1,133 @@
+.catalog {
+  --column-count: 1;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  grid-auto-flow: row;
+  padding: 50px 0;
+}
+
+@media (max-width: 488px) {
+  .catalog {
+    $column-count: 1;
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    --column-count: 4;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform $smooth;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    margin: 0 auto;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColor;
+
+    &:hover &__name {
+      color: $scale-color;
+    }
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin: 0;
+    margin-bottom: 24px;
+    font-size: 10px;
+    color: $mainColor;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-content: center;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColor;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    line-height: 18px;
+    font-size: 16px;
+    color: $mainColor;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    height: 40px;
+    background: $blue-color;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;  
+    line-height: 16px;
+    color: #fff;
+    border: 1px solid $blue-color;
+    border-radius: 5px;
+    background-color: $blue-color;
+    transition-duration: $smooth;
+
+    &__buy-button:hover {
```

still didn't fixed, please use nesting
```suggestion
    &:hover {
```